governance/votes/Executive vote - June 7, 2023.md

```diff
@@ -0,0 +1,215 @@
+- Adjust the DAI Interest Rate Strategy for Spark Protocol to match the Dai Savings Rate, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+$OfficeHours? This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### BlockTower Andromeda (RWA015-A) Onboarding
+
+As per [this poll](https://vote.makerdao.com/polling/QmbudkVR) BlockTower Andromeda will be onboarded as RWA015-A as detailed below, if this executive proposal passes.
+
+Note that the initial onboarding will use a temporarily reduced Maximum Debt Ceiling of 2.5 million DAI to ensure that the structure works as intended. The Maximum Debt Ceiling will be increased in a future executive proposal.
+
+#### Collateral Parameters
+
+- [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): **0%** - fees are calculated off-chain and returned manually.
+- [Autoline (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam): **Yes**.
+- [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam): **2.5 million DAI**.
+- [Target Availible Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): **2.5 million DAI**.
+- [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): **84,600 seconds (1 day)**.
+- [Liquidation Ratio (`mat`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): **100%**.
+- Oracle Price: **2.5 million DAI**.
+- Debt write-off timelock (`tau`): **0**.
+- Liquidation Process: Will use the established RWA collateral contracts. See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
+- [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement): QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN - [view here](https://gateway.pinata.cloud/ipfs/QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN).
+
+#### Relevant Smart Contract Information
+
+- **MIP21_LIQUIDATION_ORACLE:** `0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30`. 
+- **RWA_TOKEN_FAB:** `0x2B3a4c18705e99bC29b22222dA7E10b643658552`.
+- **SYMBOL:** `RWA015`.
+- **NAME:** `RWA-015`.
+- **ILK:** `RWA015-A`.
+- **RWA014:** `TBD`
```

Comment:
Whut? Why RWA014? This doesn't make sense as a key or a value.

Reply:
Ctrl+c, Ctrl+v error, will have the address of the RWA015 token here